include/deal.II/base/aligned_vector.h

```diff
@@ -670,6 +685,11 @@ class AlignedVector
     const AlignedVector<T> *owning_aligned_vector;
   };
 
+  /**
+   * Flag indicating if replicate_across_communicator() has been called.
+   */
+  bool replicated_across_communicator;
```

Comment:
Should we guard this with ifdef. Similar as we do in FEEval?

Reply:
In `FEEvaluationData` we don't guard the declaration of the variables but all occurrences in definitions (also in asserts, initialization in constructors). Done.

Reply:
Odd that the compiler does not warn that the variable is not initialized.

Reply:
I would leave it like this if the compiler does not complain, otherwise, the size of the vector is different for release and debug because of the additional member.